docs/root/start/sandboxes/ratelimit.rst

```diff
@@ -62,17 +66,18 @@ Use ``curl`` to make a request four times for unlimited statistics on port ``990
 
 .. code-block:: console
 
-    $ for i in {1..4}; do curl -si localhost:9901/stats/prometheus | grep "x-local-rate-limit"; done
+    $ for i in {1..5}; do curl -si localhost:9901/stats/prometheus | grep "x-local-rate-limit"; done
 
 Now, use ``curl`` to make a request four times for the limited statistics:
 
 .. code-block:: console
 
-    $ for i in {1..4}; do curl -si localhost:9902/stats/prometheus | grep "x-local-rate-limit"; done
+    $ for i in {1..5}; do curl -si localhost:9902/stats/prometheus | grep "x-local-rate-limit"; done
+    x-local-rate-limit: true
     x-local-rate-limit: true
     x-local-rate-limit: true
 
-The second and third request are limited as configured, which is expected.
+The first two requests get responses, and the rest requests are refused as expected.
```

Comment:
how are they refused ? what returncode does end user receive eg ?

Reply:
Return HTTP code 429 and "local_rate_limited" response.